static/js/publisher/metrics/metrics.js

```diff
@@ -26,13 +23,37 @@ function renderMetrics(metrics) {
     });
   });
 
-  activeDevicesMetrics(
-    metrics.activeDevices.selector,
-    activeDevices,
-    metrics.activeDevices.type,
-    metrics.defaultTrack,
-    metrics.activeDevices.annotations
-  );
+  const graph = new Graph(metrics.activeDevices.selector, activeDevices, {
+    stacked: true,
+    area: true,
+    graphType: metrics.activeDevices.type,
+    defaultTrack: metrics.defaultTrack,
+    annotations: metrics.activeDevices.annotations
+  })
+    .prepareData()
```

Comment:
Is `prepareData` optional? Can it not be called, or called in different order?
If that's a necessary step of rendering graph (that doesn't take any additional params) why not just do it automatically in constructor or internally before `render`? Does dev need to call it intentionally?

Reply:
Is this needed still? Seems like graph constructor calls it if there is any data(?)

Reply:
Ah thanks, missed that